lesson31/src/js/drawer-menu.js

```diff
@@ -8,5 +7,47 @@ const toggleDrawerMenu = () => {
   body.classList.toggle("drawer-menu-open");
 }
 
+const createOverLay = () => {
+  const overLay = document.createElement("div");
+  overLay.id = "js-drawer-overlay";
+  overLay.classList.add("drawer-overlay");
+  return overLay;
+}
+
+const renderOverLay = () => {
+  const overLay = createOverLay();
+  overLay.addEventListener("click", toggleDrawerMenu);
+  body.appendChild(overLay);
+}
+
+const switchDrawerDirect = (option) => {
+  switch (option) {
+    case "left":
+      {
+        drawerMenu.classList.add("drawer--left");
+        break;
+      }
+    case "right":
+      {
+        drawerMenu.classList.add("drawer--right");
+        break;
+      }
+    default:
+      drawerMenu.classList.add("drawer--left");
+  }
```

Comment:
```suggestion
    case "right":
      {
        drawerMenu.classList.add("drawer--right");
        break;
      }
    default:
      drawerMenu.classList.add("drawer--left");
  }
```

Reply:
Thank you !!!!!
As you said, the default is left, so I thought the code you provided was more concise.
Fixed.
[642f1f9](https://github.com/sae-github/JS_lesson/pull/39/commits/642f1f94f6babb8e7b238a6ba0be598d33979a35)